caracal/workers/transform_worker.py

```diff
@@ -52,8 +55,12 @@ def get_fields_to_split(config, name):
         if diff:
             raise caracal.ConfigurationError(
                 "'{}: field: expected 'target', 'calibrators', or one or more of {}. Got '{}'".format(name,
-                    ', '.join([f"'{f}'" for f in _cal_fields]), ','.join(diff)
-                ))
+                                                                                                      ', '.join(
```

Comment:
Is this level of indentation and line breaking OK?

Reply:
Oh dear. That's some PEP-8 formatting tool gone mad. This is what the Rise of the Machines will lead to...

Reply:
@paoloserra, I tried to fix it. Check the last commit!

Reply:
Thanks